### DISCUSSION TITLE
Prefix and suffix placeholders for leave/join messages

### DISCUSSION THREAD

**Discussion by AggerTV** [EssentialsX/Essentials#4076] [Ideas and Feature Suggestions] [answered]
I would like for there to be a suffix and prefix placeholder to use in the custom-join and custom-leave messages, so that you can format it differently from {PLAYER}.
e.g
`custom-join-message: "&8{&a+&8} {SUFFIX}{PREFIX}{USERNAME}"
custom-quit-message: "&8{&c-&8} {SUFFIX}{PREFIX}{{USERNAME}"`

**Comment by Redstylt** [comment]
Hi,

Totally agree with this suggestion !!!

**Comment by Octol1ttle** [comment, marked as answer]
This request is implemented by 887772a, please download the latest dev build from https://essentialsx.net/downloads.html to use this feature
